sockeye/inference.py

```diff
@@ -1025,6 +1140,9 @@ def __init__(self,
         self.source_with_eos = models[0].source_with_eos
         self.interpolation_func = self._get_interpolation_func(ensemble_mode)
         self.beam_size = self.models[0].beam_size
+        self.nbest_size = nbest_size
```

Comment:
Should we add a check that if `nbest_size > 1` then `beam_search_stop` should be turned off? Otherwise one could get incomplete translations in the n-best list. Alternatively we could just log a warning.

Reply:
added in translate.py where another `nbest_size > 1` already exists.